help_function/src/onedpl_test_group_load.cpp

```diff
@@ -80,19 +74,39 @@ bool test_load_subgroup_striped_standalone() {
           dpct::group::uninitialized_load_subgroup_striped<128>(item, d, thread_data);
           // Write thread_data of each work item at index to the global buffer
           int global_index = item.get_global_linear_id() * 4; // Each thread_data has 4 elements
-            for (int i = 0; i < 4; ++i) {
-                buffer[global_index + i] = thread_data[i];
+          for (int i = 0; i < 4; ++i) {
+                dacc[global_index + i] = thread_data[i];
             }
         });
   });
   q.wait_and_throw();
 
   sycl::host_accessor data_accessor(buffer, sycl::read_only);
   const int *ptr = data_accessor.get_multi_ptr<sycl::access::decorated::yes>();
-  return helper_validation_function(ptr);
+  int expected[512];
+  for (int i = 0; i < 128; i++) {
+    expected[4 * i + 0] = i;
+    expected[4 * i + 1] = 4 * i + 1;
+    expected[4 * i + 2] = 4 * i + 2;
+    expected[4 * i + 3] = 4 * i + 3;

```

Comment:
It seems that you are expecting an odd sequence and inconsistent here. 
Please check this again.